integration-cli/docker_cli_nat_test.go

```diff
@@ -75,7 +76,13 @@ func (s *DockerSuite) TestNetworkNat(c *check.C) {
 	}
 
 	result := getContainerLogs(c, srv)
-	if expected := "hello world"; result != expected {
+
+	// Ideally we'd like to check for "hello world" but sometimes
+	// nc doesn't show the data it received so instead let's look for
+	// the output of the 'echo bye' that should be printed once
+	// the nc command gets a connection
+	expected := "bye"
+	if !strings.Contains(result, expected) {
```

Comment:
@duglin IMO, we should care just about the connectivity and not the payload. By checking for "bye" we are more dependent on nc as a utility and any change in behavior in that utility will affect us again.
So, I would propose to not depend on payload comparison and just depend on if the connectivity test passes.